### PR TITLE
update app_manager urls

### PR DESCRIPTION
**Pull request by nickpell**

### CODE REVIEW

File: corehq/apps/app_manager/urls.py [hunk 1]
```diff
@@ -7,153 +7,168 @@
...
-    url(r'^copy/$', 'view_app', name='app_copy'),
-    url(r'^delete/$', 'view_app', name='app_delete'),
-    url(r'^$', 'view_app', name='view_app'),
-    url(r'^releases/$', 'view_app', name='release_manager'),
-    url(r'^releases_ajax/$', 'releases_ajax', name='release_manager_ajax'),
-    url(r'^current_version/$', 'current_app_version', name='current_app_version'),
-    url(r'^releases/json/$', 'paginate_releases', name='paginate_releases'),
-    url(r'^releases/release/(?P<saved_app_id>[\w-]+)/$', 'release_build',
+    url(r'^languages/$', view_app, name='app_languages'),
+    url(r'^languages/translations/download/$', download_bulk_ui_translations, name='download_bulk_ui_translations'),
```
Comment (dimagimon) [inline corehq/apps/app_manager/urls.py]: E501 line too long (116 > 115 characters)

File: corehq/apps/app_manager/urls.py [hunk 2]
```diff
@@ -7,153 +7,168 @@
...
+    url(r'^languages/translations/upload/$', upload_bulk_ui_translations, name='upload_bulk_ui_translations'),
+    url(r'^languages/bulk_app_translations/download/$', download_bulk_app_translations, name='download_bulk_app_translations'),
+    url(r'^languages/bulk_app_translations/upload/$', upload_bulk_app_translations, name='upload_bulk_app_translations'),
```
Comment (dimagimon) [inline corehq/apps/app_manager/urls.py]: E501 line too long (121 > 115 characters)

File: corehq/apps/app_manager/urls.py [hunk 3]
```diff
@@ -8,94 +8,108 @@
...
-    url(r'^languages/bulk_app_translations/download/$', 'download_bulk_app_translations', name='download_bulk_app_translations'),
-    url(r'^languages/bulk_app_translations/upload/$', 'upload_bulk_app_translations', name='upload_bulk_app_translations'),
-    url(r'^multimedia/$', 'view_app', name='app_multimedia'),
-    url(r'^multimedia_ajax/$', 'multimedia_ajax', name='app_multimedia_ajax'),
-    url(r'^copy/$', 'view_app', name='app_copy'),
-    url(r'^delete/$', 'view_app', name='app_delete'),
-    url(r'^$', 'view_app', name='view_app'),
-    url(r'^releases/$', 'view_app', name='release_manager'),
+    url(r'^languages/$', view_app, name='app_languages'),
+    url(r'^languages/translations/download/$', download_bulk_ui_translations, name='download_bulk_ui_translations'),
```
Comment (dimagimon) [inline corehq/apps/app_manager/urls.py]: E501 line too long (116 > 115 characters)

File: corehq/apps/app_manager/urls.py [hunk 4]
```diff
@@ -8,94 +8,108 @@
...
+    url(r'^languages/translations/upload/$', upload_bulk_ui_translations, name='upload_bulk_ui_translations'),
+    url(r'^languages/bulk_app_translations/download/$', download_bulk_app_translations, name='download_bulk_app_translations'),
```
Comment (dimagimon) [inline corehq/apps/app_manager/urls.py]: E501 line too long (127 > 115 characters)

**Comment by nickpell** [comment]
@millerdev

**Pull request merged**